plugins/com.google.cloud.tools.eclipse.appengine.validation.test/src/com/google/cloud/tools/eclipse/appengine/validation/AppEngineWebXmlValidatorTest.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.eclipse.appengine.validation;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.mock;
+
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.util.Stack;
+
+import javax.xml.parsers.ParserConfigurationException;
+import org.eclipse.core.resources.IMarker;
+import org.eclipse.core.resources.IResource;
+import org.eclipse.core.runtime.CoreException;
+import org.eclipse.wst.validation.ValidationResult;
+import org.eclipse.wst.validation.ValidatorMessage;
+import org.junit.Test;
+import org.xml.sax.SAXException;
+import org.xml.sax.SAXParseException;
+
+public class AppEngineWebXmlValidatorTest {
+
+  private static final String XML = "<application></application>";
+  private static final String BAD_XML = "<";
+  private static final String BAD_XML_MESSAGE =
+      "XML document structures must start and end within the same entity.";
+  private static final String ELEMENT_NAME = "application";
+  private static final String ELEMENT_MESSAGE =
+      "project ID tag not recommended";
+  private static final Charset UTF = StandardCharsets.UTF_8;
```

Comment:
Yeah, just remove this field and use `StandardCharsets.UTF_8` directly as suggested in `https://github.com/GoogleCloudPlatform/google-cloud-eclipse/pull/1339/files#r99422662`.